src/main/java/org/broadinstitute/hellbender/tools/exome/ReCapSegCaller.java

```diff
@@ -31,11 +33,12 @@
     public static final double DEFAULT_Z_SCORE_THRESHOLD = 2.0;
 
     //bounds on coverage we take as non-neutral a priori
-    public static final double NON_NEUTRAL_THRESHOLD_HIGH = 1.25;
-    public static final double NON_NEUTRAL_THRESHOLD_LOW = 0.75;
+    public static final double NON_NEUTRAL_THRESHOLD_HIGH = ParamUtils.log2(1.25);
+    public static final double NON_NEUTRAL_THRESHOLD_LOW = ParamUtils.log2(.75);
```

Comment:
1.25 and 0.75 have no inherent significance.  Originally there was a single threshold parameter (call it x) in log space.  The symmetric range [-x, x] maps to [2^(-x), 2^x] in real space, which requires two parameters to specify since it's not symmetric about zero.  Now that we're going back to log-space, we should restore this threshold to a single parameter with a range that is symmetric about zero.  (It could well be that the distribution in log space is not symmetric about zero, but *a priori* we don't know this).